# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Tyler Durden <virii@enn.lu>, 2016
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-09-14 14:52+0200\n"
"PO-Revision-Date: 2017-09-14 13:47+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Luxembourgish (http://www.transifex.com/otf/torproject/language/lb/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: lb\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:101
msgid "You must run this application as root"
msgstr "Dës Applikatioun muss als root ausgefouert ginn"

#: ../tails_installer/creator.py:147
msgid "Extracting live image to the target device..."
msgstr "Live Image op Zil Medium extrahéieren..."

#: ../tails_installer/creator.py:154
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Mat %(speed)d MB/sek op d'Medium geschriwwen"

#: ../tails_installer/creator.py:184
msgid "Setting up OLPC boot file..."
msgstr "OLPC boot Datei gëtt ageriicht..."

#: ../tails_installer/creator.py:321
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Et gouf e Problem beim ausféiere vu folgendem Kommando: `%(command)s`.\nEng méi detailléiert Feelerlogdatei gouf an '%(filename)s' geschriwwen."

#: ../tails_installer/creator.py:340
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "SHA1 Checkzomm vun der LiveCD verifizéieren..."

#: ../tails_installer/creator.py:344
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "SHA256 Checkzomm vun der LiveCD verifizéieren..."

#: ../tails_installer/creator.py:360
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Feeler: SHA1 Checkzomm vun der Live CD ass falsch. Dir kennt dëst Programm mat dem --noverify Argument starten em de Verifikatiounscheck ze ëmgoen."

#: ../tails_installer/creator.py:366
msgid "Unknown ISO, skipping checksum verification"
msgstr "Onbekannten ISO, Checkzomm Verifikatioun gëtt iwwersprongen"

#: ../tails_installer/creator.py:377
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Net genuch fräi Späicherplaz op dem Medium.\n%dMB ISO + %dMB Späicherplaz benéidegt > %dMB fräie Späicher"

#: ../tails_installer/creator.py:384
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "%sMB u persistenter Späicherplaz goufen erstallt."

#: ../tails_installer/creator.py:445
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Kopéiere vun %(infile)s no %(outfile)s ass schif gaangen: %(message)s"

#: ../tails_installer/creator.py:459
msgid "Removing existing Live OS"
msgstr "Läschen existéierend Live OS"

#: ../tails_installer/creator.py:468 ../tails_installer/creator.py:479
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "chmod vun %(file)s net méiglech: %(message)s"

#: ../tails_installer/creator.py:472
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Datei vum leschte LiveOS konnt net geläscht ginn: %(message)s"

#: ../tails_installer/creator.py:484
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Verzeechnes vum leschte LiveOS konnt net geläscht ginn: %(message)s"

#: ../tails_installer/creator.py:532
#, python-format
msgid "Cannot find device %s"
msgstr "Kann d'Maschinn %s net fannen"

#: ../tails_installer/creator.py:699
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Net méiglech op %(device)s ze schreiwen, iwwersprangen."

#: ../tails_installer/creator.py:719
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "E puer Partitioune vum ausgewielte Medium %(device)s si gemount. Se wäerten ausgehaange ginn ier den Installatiounsprozess ufänkt."

#: ../tails_installer/creator.py:762 ../tails_installer/creator.py:986
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Onbekanntent Dateisystem. D'Medium muss néi formatéiert ginn."

#: ../tails_installer/creator.py:765 ../tails_installer/creator.py:989
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Dateisystem net ënnerstëtzt: %s"

#: ../tails_installer/creator.py:782
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:786
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Medium konnt net agehaangen ginn: %(message)s"

#: ../tails_installer/creator.py:791
msgid "No mount points found"
msgstr "Keng Plaz zum Mounten fonnt"

#: ../tails_installer/creator.py:802
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "unmount_devide fir '%(device)s'"

#: ../tails_installer/creator.py:812
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Aushänke vum agehaangenem Dateisystem op '%(device)s'"

#: ../tails_installer/creator.py:816
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Aushänke vu '%(udi)s' op '%(device)s'"

#: ../tails_installer/creator.py:826
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Mount %s existéiert nach no dem aushänken."

#: ../tails_installer/creator.py:839
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Späichermedium %(device)s gëtt partitionéiert"

#: ../tails_installer/creator.py:917
#, python-format
msgid "Updating properties of system partition %(system_partition)s"
msgstr "Astellunge vun der System Partitioun %(system_partition)s gi geupdatet"

#: ../tails_installer/creator.py:971
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Medium net ënnerstëtzt '%(device)s'. Mellt wann ech gelift de Feeler."

#: ../tails_installer/creator.py:974
msgid "Trying to continue anyway."
msgstr "Versichen trotzdem weider ze maachen."

#: ../tails_installer/creator.py:983 ../tails_installer/creator.py:1376
msgid "Verifying filesystem..."
msgstr "Dateisystem verifizéieren..."

#: ../tails_installer/creator.py:1007
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Ännerung vun der Volumebeschreiwung net méiglech: %(message)s"

#: ../tails_installer/creator.py:1012 ../tails_installer/creator.py:1409
msgid "Installing bootloader..."
msgstr "Bootloader gëtt installéiert..."

#: ../tails_installer/creator.py:1038
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr ""

#: ../tails_installer/creator.py:1046 ../tails_installer/creator.py:1427
#, python-format
msgid "Removing %(file)s"
msgstr "Läschen %(file)s"

#: ../tails_installer/creator.py:1158
#, python-format
msgid "%s already bootable"
msgstr "%s ass schonns bootbar"

#: ../tails_installer/creator.py:1178
msgid "Unable to find partition"
msgstr "Et gouf keng Partitioun fonnt"

#: ../tails_installer/creator.py:1201
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "%(device)s gëtt als FAT32 formatéiert."

#: ../tails_installer/creator.py:1261
msgid "Could not find syslinux' gptmbr.bin"
msgstr ""

#: ../tails_installer/creator.py:1274
#, python-format
msgid "Reading extracted MBR from %s"
msgstr ""

#: ../tails_installer/creator.py:1278
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1291 ../tails_installer/creator.py:1292
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Master Boot Record vun %s gëtt zeréck gesat"

#: ../tails_installer/creator.py:1297
msgid "Drive is a loopback, skipping MBR reset"
msgstr "D'Lafwierk ass e Loopback, MBR zerécksetze gëtt iwwersprongen"

#: ../tails_installer/creator.py:1301 ../tails_installer/creator.py:1555
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "SHA1 Zomm vun %s gëtt berechent"

#: ../tails_installer/creator.py:1326
msgid "Synchronizing data on disk..."
msgstr "Daten op dem Disk gi synchroniséiert..."

#: ../tails_installer/creator.py:1368
msgid "Error probing device"
msgstr "Feeler beim iwwerpréiwe vum Späichermedium"

#: ../tails_installer/creator.py:1370
msgid "Unable to find any supported device"
msgstr "Et konnt kee Medium fonnt gi wat ënnerstëtzt gëtt"

#: ../tails_installer/creator.py:1380
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Stellt sécher, datt den USB Stick agestach ass an FAT formatéiert ass"

#: ../tails_installer/creator.py:1383
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Dateisystem net ënnerstëtzt: %s\nMacht ee Backup a formatéiert den USB Stick mat dem FAT Dateisystem."

#: ../tails_installer/creator.py:1450
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Win32_LogicalDisk konnt net ermëttelt ginn; de win32com Opruff huet keng Resultater bruecht."

#: ../tails_installer/creator.py:1503
msgid "Cannot find"
msgstr "Kann net fonnt ginn"

#: ../tails_installer/creator.py:1504
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Onbekannt release: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "%s gëtt erofgelued"

#: ../tails_installer/gui.py:209
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Feeler: Kann d'Bezeechnung oder UUID vum Medium net ermëttelen. Weiderfueren ass net méiglech."

#: ../tails_installer/gui.py:258
#, python-format
msgid "Installation complete! (%s)"
msgstr "Installatioun ofgeschloss! (%s)"

#: ../tails_installer/gui.py:263
msgid "Tails installation failed!"
msgstr ""

#: ../tails_installer/gui.py:364
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Warnung: Dëst Tool muss als Administrateur ausgefouert ginn. Fir dat ze maachen, musst der op der Maus op den Icon riets klicken an dann d'Astellungen auswielen. Ënnert dem Reider \"Kompatibilitéit\", wielt der dann \"Dëst Programm als Administrateur ausféieren\" aus."

#: ../tails_installer/gui.py:376 ../tails_installer/launcher.py:31
msgid "Tails Installer"
msgstr ""

#: ../tails_installer/gui.py:473
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:484
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:485
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:524
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:526
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:560
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:570
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "D'Medium \"%(pretty_name)s\" ass ze kleng fir Tails dorobber ze installéieren (et ginn op mannst %(size)s GB gebraucht)."

#: ../tails_installer/gui.py:592
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:604
msgid "Refreshing releases..."
msgstr "Verëffentlechunge ginn aktualiséiert."

#: ../tails_installer/gui.py:609
msgid "Releases updated!"
msgstr "Verëffentlechunge goufen aktualiséiert!"

#: ../tails_installer/gui.py:651
msgid "Installation complete!"
msgstr "Installatioun ofgeschloss!"

#: ../tails_installer/gui.py:652
msgid "Installation was completed."
msgstr ""

#: ../tails_installer/gui.py:701
msgid "Unable to mount device"
msgstr "Medium konnt net agehaange ginn"

#: ../tails_installer/gui.py:708 ../tails_installer/gui.py:736
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:709
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:728
#, python-format
msgid ""
"%(vendor)s %(model)s device (%(device)s)\n"
"\n"
"%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:782
msgid "Download complete!"
msgstr "Download fäerdeg!"

#: ../tails_installer/gui.py:786
msgid "Download failed: "
msgstr "Download feel geschloen"

#: ../tails_installer/gui.py:787
msgid "You can try again to resume your download"
msgstr "Download probéieren ze resuméieren"

#: ../tails_installer/gui.py:795
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Déi ausgewielten Datei ass net liesbar. Ännert wann ech gelift är Berechtegungen oder wielt eng aner Datei aus."

#: ../tails_installer/gui.py:801
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Net méiglech déi ausgewielten Datei ze benotzen. Dir hutt wuel méi Gléck wann dir är ISO direkt an engem Lafwierk ofleet (zB: C:\\)"

#: ../tails_installer/gui.py:807
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s ausgewielt"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr ""

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr ""

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr ""

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr ""

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr ""

#: ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr ""

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""

#: ../data/tails-installer.ui.h:7
msgid "Install Tails"
msgstr "Tails installéieren"

#: ../data/tails-installer-launcher.ui.h:1
msgid ""
"To run Tails Installer you need an ISO image which can be downloaded from "
"the Tails website: <a "
"href=\"https://tails.boum.org/download/\">https://tails.boum.org/download/</a>"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:2
msgid "Install"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:3
msgid "• Install Tails on a new USB stick."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:4
msgid "• The USB stick that you install on is formatted and all data is lost."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:5
msgid "Upgrade"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:6
msgid "• Upgrade a Tails USB stick to the version of an ISO image."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:7
msgid ""
"• The encrypted persistent storage of the Tails USB stick that you upgrade "
"is preserved."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:8
msgid ""
"Need help? Read the <a "
"href=\"https://tails.boum.org/doc/first_steps/installation/\">documentation</a>"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:9
msgid "Install by cloning"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:10
msgid ""
"• Install Tails on another USB stick by copying the Tails system that you "
"are currently using."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:11
msgid ""
"• The encrypted persistent storage of the Tails USB stick that you are "
"currently using is not copied."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:12
msgid "Upgrade by cloning"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:13
msgid ""
"• Upgrade another Tails USB stick to the same version of Tails that you are "
"currently using."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:14
msgid "Upgrade from ISO"
msgstr ""

#: ../data/tails-installer-launcher.ui.h:15
msgid "• Upgrade another Tails USB stick to the version of an ISO image."
msgstr ""

#: ../data/tails-installer-launcher.ui.h:16
msgid ""
"Need help? Read the <a "
"href=\"file:///usr/share/doc/tails/website/doc/first_steps/installation.en.html\">documentation</a>"
msgstr ""
